mgmtsystem_action/mgmtsystem_action.py

```diff
@@ -80,13 +80,8 @@ def case_open(self):
 
         for case in self:
             values = {'active': True}
-
-            stages = self.env['mgmtsystem.action.stage']
-            values['stage_id'] = stages.search([
-                ['is_ending', '=', False],
-                ['is_starting', '=', False]
-            ]).id
-
+            values['stage_id'] = self.env.ref(
+                'mgmtsystem_action.stage_open').id
```

Comment:
@dreispt I don't agree. If you add limit, you get a non-deterministic behavior.

And the name of the method is "case_open". So the logic state of the action is "open".